frontend/src/shared/components/IconBadgeList/IconBadgeList.tsx

```diff
@@ -0,0 +1,26 @@
+import { PLATFORM_ICON_MAP, TECH_ICON_MAP } from "./constants/iconMap";
+import IconBadge from "./IconBadge/IconBadge";
+import * as S from "./IconBadgeList.styled";
+
+interface IconBadgeListProps {
+  iconBadges: string[];
+}
+
+const iconBadgeMap: Record<string, { label: string; imgUrl: string }> = {
+  ...TECH_ICON_MAP,
+  ...PLATFORM_ICON_MAP,
+};
+
+function IconBadgeList({ iconBadges }: IconBadgeListProps) {
+  return (
+    <S.IconBadgeList>
+      {iconBadges.map((key) => {
+        const data = iconBadgeMap[key.toLowerCase()];
+        if (!data) return null;
+        return <IconBadge key={key} label={data.label} imgUrl={data.imgUrl} />;
+      })}
```

Comment:
```suggestion
      {iconBadges.map((key) => {
        const { label, imgUrl } = iconBadgeMap[key.toLowerCase()];
        
        return <IconBadge key={key} label={label} imgUrl={imgUrl} />;
      })}
```

`iconBadges`의 타입을 잘 잡으면 `data`가 없는 경우는 고려하지 않아도 될 것 같네요🙃

추가적으로 `key.toLowerCase()`를 해줘야 하는 이유도 궁금합니다~